source/extensions/filters/http/cache/cache_filter.h

```diff
@@ -38,19 +39,50 @@ class CacheFilter : public Http::PassThroughFilter,
   Http::FilterDataStatus encodeData(Buffer::Instance& buffer, bool end_stream) override;
 
 private:
+  // Utility functions; make any necessary checks and call the corresponding lookup_ functions
```

Comment:
nit: We need to start the comment with capital end a sentence with a period. It applies anywhere else.